host.go

```diff
@@ -123,11 +123,11 @@ func (config *OtelConfig) resolveSignalUrl(signal otelSignal) string {
 }
 
 func (config *OtelConfig) defaultEndpoint() string {
-	endpoint := OtelExporterHttpEndpoint
 	if config.OtelProtocol() == OtelProtocolGRPC {
-		endpoint = OtelExporterGrpcEndpoint
+		return OtelExporterGrpcEndpoint
 	}
-	return endpoint
+
+	return OtelExporterHttpEndpoint
```

Comment:
is `else {}` not a common pattern here in Go?

Reply:
In cases like this, [it's not](https://go.dev/doc/effective_go#else).